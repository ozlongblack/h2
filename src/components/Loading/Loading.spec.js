// @flow

import React from 'react';
import renderer from 'react-test-renderer';
import LoadingView from './Loading.view';

it('renders correctly', () => {
  const tree: JSON = renderer
    .create(<LoadingView />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
