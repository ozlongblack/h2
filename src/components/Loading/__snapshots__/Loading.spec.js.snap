// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="loading loading--inactive"
>
  <div
    className="loading__screen"
  >
    <div
      className="loading__spinner"
    >
      <svg
        className="loading__circular"
      >
        <circle
          className="loading__circular__path"
          cx="60"
          cy="60"
          fill="none"
          r="20"
          strokeMiterlimit="10"
          strokeWidth="2"
        />
      </svg>
    </div>
  </div>
</div>
`;
